src/main/java/org/retest/rebazer/connector/GithubConnector.java

```diff
@@ -28,17 +29,23 @@ public GithubConnector( final RepositoryConfig repoConfig, final RestTemplateBui
 
 		template = builder.basicAuthentication( repoConfig.getUser(), repoConfig.getPass() )
 				.rootUri( BASE_URL + basePath ).build();
+
 	}
 
 	@Override
 	public PullRequest getLatestUpdate( final PullRequest pullRequest ) {
 		final DocumentContext jsonPath = jsonPathForPath( requestPath( pullRequest ) );
-		return PullRequest.builder() //
+		final String repositoryTime = jsonPath.read( "$.updated_at" );
+		final String checksTime = newestChecksTime( pullRequest );
+		final int solution = repositoryTime.compareTo( checksTime );
```

Comment:
compareTo only works, because of The ISO 8601 standard (yyyy-mm-ddThh:mm:ssZ)

Reply:
But not if the time zone differs, I would prefere to parse the dates and compare these.

Reply:
sry, I should have noticed that "updated_at" could be in a different time zone.